src/server/controllers/users.controller.js

```diff
@@ -598,6 +607,70 @@ export function getMentoringRequestStatus(req, res, next) {
     });
 }
 
+export function activeBookmark(req, res, next) {
+  User.findOne({ _id: req.user._id }).exec()
+    .then((user) => {
+      user.bookmark.push(req.body.id);
+      user.save();
+    })
+    .then(() => {
+      res.status(200).json({ msg: userCallback.SUCCESS_ACTIVE_BOOKMARK });
+    })
+    .catch((err) => {
+      res.status(400).json(err);
+    });
+}
+
+export function inActiveBookmark(req, res, next) {
+  User.findOne({ _id: req.user._id }).exec()
+    .then((user) => {
+      const index = user.bookmark.indexOf(req.body.id);
+      user.bookmark.splice(index, 1);
+      user.save();
+    })
+    .then(() => {
+      res.status(200).json({ msg: userCallback.SUCCESS_INACTIVE_BOOKMARK });
+    })
+    .catch((err) => {
+      res.status(400).json(err);
+    });
+}
+
+export function getBookmark(req, res, next) {
+  User.findOne({ _id: req.user._id }).exec()
+    .then((user) => {
+      let bookmarkList = user.bookmark;
+      return User.find({
+        _id: {
+          $in: bookmarkList,
+        },
+      }, { password: 0 }).exec();
```

Comment:
This is for **not containing password** in data to send. Even other sending mentorlist or sending user profile contain password information, and this should be changed later.